haas/api.py

```diff
@@ -773,32 +773,22 @@ def list_project_networks(project):
 
 @rest_call('GET', '/node/<nodename>')
 def show_node(nodename):
-    """Show details of a node.
-
-    Returns a JSON object representing a node.
-    The object will have at least the following fields:
-        * "name", the name/label of the node (string).
-        * "free", indicates whether the node is free or has been allocated
-            to a project.
-        * "nics", a list of nics, each represted by a JSON object having
-            at least the following fields:
-                - "label", the nic's label.
-                - "macaddr", the nic's mac address.
-
-    Example:  '{"name": "node1",
-                "free": True,
-                "nics": [{"label": "nic1", "macaddr": "01:23:45:67:89"},
-                         {"label": "nic2", "macaddr": "12:34:56:78:90"}]
-               }'
-    """
-    
+    '''
```

Comment:
Please follow [PEP257](https://www.python.org/dev/peps/pep-0257/) here, specifically:
 * Using double-quotes
 * Either using a one-liner, or following the conventions for where lines start/stop.